Doc about migrating from v1beta1 to v1 (#5415)
tektoncd/pipeline: docs/pipelineresources.md

## Replacing `PipelineResources` with `Tasks`

> `PipelineResources` remained in alpha while the other resource kinds were promoted to beta.
> Since then, **`PipelineResources` have been deprecated**. We encourage users to use `Tasks` and other replacement 

Reviewer: ```suggestion
**`PipelineResources` are deprecated**. We encourage users to use `Tasks` ... 
```
